src/index.js

```diff
@@ -1,17 +1,17 @@
 'use strict'
 const core = require('@actions/core')
 const toolkit = require('actions-toolkit')
-const { staleDaysToMs } = require('./time-utils.js')
+const { parseNotificationSettings } = require('./time-utils.js')
 const { runAction } = require('./release-notify-action')
 
 async function run() {
   toolkit.logActionRefWarning()
 
   const token = core.getInput('github-token', { required: true })
-  const staleDate = staleDaysToMs(core.getInput('stale-days'))
+  const notifyAfter = parseNotificationSettings(core)
```

Comment:
this function's signature should be improved:

- naming: now it's only giving you a single value, so let's make sure the name of the function reflects what it's doing. Also let's make sure it's clear what the return value is and reflect that in the variable we assign it to. At the moment "notifyAfter" doesn't really tell us much about what it contains
- signature: since we're making this function aware of the inputs, no point in providing it the core library, since it can just import it itself